public/locales/en/translation.json

```diff
@@ -0,0 +1,7 @@
+{
+    "title-app":"todoapp&react",
+    "Tasks": "Tasks",
+    "Finished tasks": "Finished tasks",
+    "Add task": "Add task"
+    
+}
```

Comment:
If I 'm not wrong, and this is your dictionary, you should keep your keys in camelcase notation